domains/krix.json

```diff
@@ -5,11 +5,6 @@
         "email": "kristjan.kocevar89@gmail.com"
       },
     "record": {
-        "A": ["35.186.245.55"],
-        "MX": [
-            "mx1.improvmx.com",
-            "mx2.improvmx.com"
-        ],
-        "TXT": "v=spf1 include:spf.improvmx.com ~all"
+        "CNAME": "krix12.github.io"
```

Comment:
i think i've got 404?

Reply:
Should I then replace it with https://krix12.github.io/Krix-portfolio/?

Reply:
Fixed now